### PR TITLE
Update ACM CRD for v1.16.0

### PR DESCRIPTION
**Pull request by karlkfi**
Do not merge until 2023-08-17

### CODE REVIEW

**Review by janetkuo — COMMENTED** [review]
/lgtm
/hold

File: crds/configmanagement_v1_configmanagement.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2023 Google LLC
```
Comment (bourgeoisor) [inline crds/configmanagement_v1_configmanagement.yaml:1]: According to go/copyright we shouldn't change this / need to retain the earliest year that appeared in the file

Comment (karlkfi) [reply crds/configmanagement_v1_configmanagement.yaml]: reverted

**Review by bourgeoisor — APPROVED** [review]

**Comment by karlkfi** [comment]
Released delayed until 2023-08-14

**Comment by bourgeoisor** [comment]
> Released delayed until 2023-08-14

Is that a typo? August 14th was a few days ago

**Comment by karlkfi** [comment]
Yea. 2023-08-21 Monday

**Pull request merged**